hack/deploy_fake_gpu_operator.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+#
+# Deploy https://github.com/run-ai/fake-gpu-operator
+#
+# This is useful to simulate nvidia GPU resources locally.
+
+set -euo pipefail
+
+kubectl get nodes -o name | xargs -I {} kubectl label {} nvidia.com/gpu.deploy.device-plugin=true nvidia.com/gpu.deploy.dcgm-exporter=true --overwrite
```

Comment:
Not sure but I think it's safe to quote curly braces?  `'{}'`

Reply:
Done!
```suggestion
kubectl get nodes -o name | xargs -I '{}' kubectl label '{}' nvidia.com/gpu.deploy.device-plugin=true nvidia.com/gpu.deploy.dcgm-exporter=true --overwrite
```